src/main/java/uk/gov/pay/directdebit/mandate/api/GetMandateResponse.java

```diff
@@ -16,38 +18,38 @@
 @JsonInclude(Include.NON_NULL)
 @JsonFormat(shape = JsonFormat.Shape.OBJECT)
 public class GetMandateResponse {
+
+    @JsonProperty("provider_id")
+    private final String paymentProviderId;
```

Comment:
I think this should use the PaymentProviderPaymentId type with a `ToStringSerializer`

Reply:
Thanks for reviewing. I've fixed this.